framework-tests/app/build.gradle

```diff
@@ -1,12 +1,5 @@
 apply plugin: 'com.android.application'
 
```

Comment:
There is a need for a common gradle file but I will take care of it after things settle down.